ihs/src/main/scripts/install.sh

```diff
@@ -0,0 +1,94 @@
+#!/bin/sh
+
+#      Copyright (c) Microsoft Corporation.
+# 
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+# 
+#           http://www.apache.org/licenses/LICENSE-2.0
+# 
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+while getopts "u:p:" opt; do
+    case $opt in
+        u)
+            userName=$OPTARG #IBM user id for downloading artifacts from IBM web site
+        ;;
+        p)
+            password=$OPTARG #password of IBM user id for downloading artifacts from IBM web site
+        ;;
+    esac
+done
+
+# Wait untile the data disk is partitioned and mounted
+output=$(df -h)
+while echo $output | grep -qv "/datadrive"
+do
+    sleep 10
+    echo "Waiting for data disk partition & moute complete..."
```

Comment:
moute -> did you mean `mount`?

Reply:
@git4rk  I have addressed your comments in commit 81bd04d.